components/Form/FormQuantitySpecification/index.js

```diff
@@ -0,0 +1,142 @@
+import styled from "styled-components";
+import SubmitButton from "../../Buttons/SubmitButton/SubmitButton";
+import ResetFormAndResultButton from "../../Buttons/ResetButton/ResetFormAndResultButton";
+import { BasicButton } from "../../Buttons/buttonStyles";
+import Link from "next/link";
+import { useState } from "react";
+import {
+  calculateDetailsScoreDeviation,
+  calculateDetailsScoreAccordance,
+} from "../../../utils/detailsScoreUtils";
+
+export default function FormQuantitySpecification({ foodCategory }) {
+  const [showResult, setShowResult] = useState(false);
+  const [resultText, setResultText] = useState("");
+  const [quantity, setQuantity] = useState(0);
+
+  const {
+    id,
+    name,
+    recommendedConsumption,
+    recommendedExample,
+    maxRange,
+    maxRangeInputField,
+  } = foodCategory;
+
+  function handleQuantitySpecification(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    setQuantity(parseFloat(data.consumedQuantity));
+    const consumedQuantityAccordance = calculateDetailsScoreAccordance(
+      calculateDetailsScoreDeviation,
+      quantity,
+      recommendedConsumption,
+      maxRange
+    );
+    const result = `Dein Konsum der Lebensmittelgruppe ${name} stimmt zu ${Math.floor(
+      consumedQuantityAccordance
+    )} % mit den Empfehlungen der Planetary Health Diet überein. Empfohlen werden ${recommendedConsumption} gramm. Das entspricht etwa: ${recommendedExample}.`;
+    setResultText(result);
+    setShowResult(true);
+    event.target.reset();
+  }
+
+  return (
+    <>
+      <Form
+        aria-labelledby="title"
```

Comment:
Doesn't match the id of your `h3`, should be `form-title`

Reply:
Implemented that, thanks! :-)